docs/modules/ROOT/pages/how-tos/upgrade-1.1.1-1.7.x.adoc

```diff
@@ -0,0 +1,9 @@
+= Upgrade from 1.1.x to 1.2.x
```

Comment:
The new component version *must* be v2.0.0 since we consider this PR a breaking change.

```suggestion
= Upgrade from v1.x to v2.x
```

Reply:
Wasn't sure about that when I was writing the tutorial